pythonfuzz/fuzzer.py

```diff
@@ -125,8 +127,13 @@ def start(self):
         self._p = mp.Process(target=worker, args=(self._target, child_conn, self._close_fd_mask))
         self._p.start()
 
-
         while True:
+            if self.runs != -1:
+                if self._total_executions == self.runs:
```

Comment:
I would use an `and` instead of nested `if` and also would use `self._total_executions >= self.runs`. other then that looks good. thx!